src/webkit/WebKit_Startup_002.test.js

```diff
@@ -0,0 +1,45 @@
+import {
+  pluginDeactivate,
+  checkIfProcessIsRunning,
+  restartFramework,
+  webKitBrowserOps,
+  stopWPEFramework,
+  startFramework,
+} from '../commonMethods/commonFunctions'
+import constants from '../commonMethods/constants'
+
+export default {
+  title: 'WPEWebkit shutdown of Framework robustness test',
+  description: 'Starts WPEWebkit and stops Framework. Checks if everything is shutdown correctly',
+  repeat: 30,
+  setup() {
+    return this.$sequence([
+      () => pluginDeactivate.call(this, constants.youTubePlugin),
+      () => pluginDeactivate.call(this, constants.netFlixPlugin),
+    ])
+  },
+  teardown: restartFramework,
+  steps: [
+    {
+      description: 'Activate WebKit Plugin and check whether it is resumed correctly',
+      test: webKitBrowserOps,
```

Comment:
Already commented on other PR as well, this should be renamed to something more concrete. Like webKitBrowserRestart or webKitBrowserRestartAndResume.

Reply:
Renamed method name to a better one.